### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by wynged**
BACKGROUND:
- #485  didn't update changelog with some additional changes

---
This change is [Reviewable](https://reviewable.io/reviews/hypar-io/elements/486)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
 # Changelog
 
 ## 0.8.2
+### Changed
+- Model deserialization errors are now and `out` variable not a list you pass in.
```
Comment (ikeough) [inline CHANGELOG.md:5]: I suggest: The errors parameter for Model.FromJson now has the `out` modifier. It no longer takes a default value.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
+- Model deserialization only refreshes type cache if forceTypeReload is set to true.
```
Comment (ikeough) [inline CHANGELOG.md:6]: " ...if the `forceTypeReload` parameter is set to true"

**Review by ikeough — APPROVED** [review]
Approved with small modifications.

**Review by wynged — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/hypar-io/elements/486)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @ikeough)

___
*[CHANGELOG.md, line 5 at r1](https://reviewable.io/reviews/hypar-io/elements/486#-MRW9rquT4tqcJyNq0m3-r1-5:-MRWd3Yk9MQrSPq3GNcz:b-896fix) ([raw file](https://github.com/hypar-io/elements/blob/2f62b42542a473d7f7fbf8f223539b8cea72efe4/CHANGELOG.md#L5)):*
<details><summary><i>Previously, ikeough (Ian Keough) wrote…</i></summary><blockquote>

I suggest: The errors parameter for Model.FromJson now has the `out` modifier. It no longer takes a default value.
</blockquote></details>

Done.
___
*[CHANGELOG.md, line 6 at r1](https://reviewable.io/reviews/hypar-io/elements/486#-MRW9rquT4tqcJyNq0m3-r1-6:-MRWd3yT6XQkaAfzsBIh:b-896fix) ([raw file](https://github.com/hypar-io/elements/blob/2f62b42542a473d7f7fbf8f223539b8cea72efe4/CHANGELOG.md#L6)):*
<details><summary><i>Previously, ikeough (Ian Keough) wrote…</i></summary><blockquote>

" ...if the `forceTypeReload` parameter is set to true"
</blockquote></details>

Done.

**Pull request merged**